### PR TITLE
Uncommenting test

### PR DESCRIPTION
**Pull request by twokul**
fixes https://github.com/ember-cli/ember-cli/issues/5079

### CODE REVIEW

**Comment by Turbo87** [comment]
looks like the test is actually failing for some reason

**Comment by twokul** [comment]
@Turbo87 yeah, looking into it

File: tests/acceptance/smoke-test-slow.js [hunk 1]
```diff
@@ -119,19 +119,18 @@ describe('Acceptance: smoke-test', function() {
...
-  //         })
-  //         .catch(function() {
-  //           expect(false, 'should NOT have rejected with a failing test');
-  //         });
-  //     });
-  // });
+  it('ember test exits with zero when tests pass', function() {
+    return copyFixtureFiles('smoke-tests/passing-test')
+      .then(function() {
+        return runCommand(path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember'), 'test')
```
Comment (Turbo87) [inline tests/acceptance/smoke-test-slow.js]: why is `ember(['test'])` not working correctly here?

**Comment by stefanpenner** [comment]
The point of the ember helper is to invoke programmatically. The test was uncommented so we would eventually find out what underlying bug is preventing it from working. Changing this to be just an exec, side-steps the goal.

**Comment by nathanhammond** [comment]
We have something that is tremendously trolling us here and shows up in multiple issues. I've spent days trying to track it down to no avail, worked with @stefanpenner to no avail, and I've run out of energy to invest in this. This is likely the same thing that is blocking the multi meta modules PRs.

**Comment by stefanpenner** [comment]
Good job tracking this done, i believe though we should fix this in testem.

**Comment by rwjblue** [comment]
I'm unsure if this is anything but a test concern. Normally, we would want to follow normal require rules for testem.js (and we do not support changes to testem.js at runtime and restart anyways). 

It seems completely reasonable for testem to be requiring testem.js (FWIW we also require ember-cli-build.js exactly the same way as testem is requiring its own configuration).

**Comment by stefanpenner** [comment]
> I'm unsure if this is anything but a test concern.

yes, but it would be better if we didn't have to remember in each test that potentially hits it. IMHO

**Comment by rwjblue** [comment]
Confirm. Maybe we can clear in a general place that is fired for each test (for example: perhaps the ember helper could do this on cleanup when the command is 'test').

**Comment by stefanpenner** [comment]
@rwjblue not a bad idea

**Comment by twokul** [comment]
@rwjblue @stefanpenner I can change `ember` helper then

**Comment by stefanpenner** [comment]
@twokul sounds good, thanks @rwjblue for the idea

**Comment by Turbo87** [comment]
@stefanpenner @rwjblue r?

**Comment by stefanpenner** [comment]
@homu r+

**Comment by stefanpenner** [comment]
good catch @twokul

**Pull request merged**